depends/patches/qt/freetype_back_compat.patch

```diff
@@ -0,0 +1,28 @@
+commit 14bc77db61bf9d56f9b6c8b84aa02573605c19c6
+Author: fanquake <fanquake@gmail.com>
+Date:   Tue Aug 18 15:15:08 2020 +0800
+
+    Fix backwards compatibility with older Freetype versions at runtime
+
+    A few years ago, libfreetype introduced FT_Get_Font_Format() as an alias
+    for FT_Get_X11_Font_Format(), but FT_Get_X11_Font_Format() was kept for abi
+    backwards-compatibility.
+
+    Qt 5.9 introduced a call to FT_Get_Font_Format(). Replace it with FT_Get_X11_Font_Format()
+    in order to remain compatibile with older freetype, which is still used by e.g. Ubuntu Trusty.
+
+    See 14348.
```

Comment:
e2b18a4120f344d2043329f0382e77e7fe9ae933
nano-nit:
```suggestion
    See #14348.
```

Reply:
Will be fixed in the next push.